codestyle/checkstyle.xml

```diff
@@ -204,6 +204,12 @@
       <property name="message" value="Use org.apache.druid.server.http.HttpMediaType#TEXT_PLAIN_UTF8 instead"/>
     </module>
 
+    <module name="Regexp">
+      <property name="format" value='^((?!throw|=).)* Throwables.propagate\('/>
```

Comment:
https://github.com/apache/incubator-druid/pull/6939#discussion_r253419677 not addressed.